NOTICE.txt

```diff
@@ -230,27 +261,19 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 --------------------------------------------------------------------
 Dependency: github.com/aws/aws-sdk-go-v2
 Version: v0.9.0
-Revision: 098e15df3044cf1b04a222c1c33c3e6135ac89f3
 License type (autodetected): Apache-2.0
 ./vendor/github.com/aws/aws-sdk-go-v2/LICENSE.txt:
 --------------------------------------------------------------------
 Apache License 2.0
 
-
---------------------------------------------------------------------
-Dependency: github.com/awslabs/goformation
-Version: v4.1.0
-Revision: 18ccd0e58e5df10d9f82f875d54b087d3058cfc1
-License type (autodetected): Apache-2.0
-./vendor/github.com/awslabs/goformation/LICENSE:
---------------------------------------------------------------------
-Apache License 2.0
-
+-------NOTICE.txt-----
+AWS SDK for Go
+Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+Copyright 2014-2015 Stripe, Inc.
```

Comment:
Is there a reason we want to include these lines?

Reply:
The notice generator script adds NOTICE.txt files of Apache licensed dependencies. I dug into the git history of the file and found the following comment (`git show e2aaa38a8e9a7fe382aacb8d3c2e5074af20f43a`):

> For each 3rd party library, the following rules are used:
> - If the library has apache license, and if NOTICE file is present, include that text in  the NOTICE file.
> - If the library has MIT, ISC or BSD license, include the whole copyright block in NOTICE file (meaning copyright + license)